src/components/job-board/jobBoard.tsx

```diff
@@ -14,73 +17,139 @@ export default function JobBoard() {
-                  <JobDropdown />
-                </CardHeader>
-                <CardContent className="space-y-1 text-sm text-gray-700">
-                  <p className="font-medium text-gray-900">{job.company}</p>
-                  <p className="text-gray-600">{job.location}</p>
-                  <p className="font-medium text-orange-600">{job.pay}</p>
-                  <div className="mt-2 flex flex-wrap gap-2">
-                    {job.tags.map((tag) => (
-                      <span
-                        key={tag.id}
-                        className="rounded-lg bg-blue-100 px-1 py-1 text-xs font-medium text-blue-600"
-                      >
-                        {tag.name}
-                      </span>
-                    ))}
-                  </div>
-                </CardContent>
-              </Card>
-            ))}
-          </div>
+          <ScrollArea className="w-1/3 border-r border-[#FF5331] pr-6">
+            <div className="space-y-4">
+              {jobCards.map((job) => (
+                <Card
+                  key={job.id}
+                  onClick={() => setSelectedJob(job)}
+                  className={`cursor-pointer rounded-xl border border-[#FF5331] p-5 transition-all hover:border-[#FF5331] hover:shadow-lg ${
+                    selectedJob?.id === job.id ? 'border-[#FF5331] bg-[#FFF5F3]' : ''
+                  }`}
+                >
+                  <CardHeader className="flex flex-row items-start justify-between space-y-0 pb-3">
+                    <div className="space-y-1.5">
+                      <CardTitle className="text-lg font-semibold text-gray-900 hover:text-[#FF5331]">
+                        {job.title}
+                      </CardTitle>
+                      <div className="flex items-center gap-2 text-sm text-gray-600">
+                        <Building2 className="h-4 w-4" />
+                        <span className="font-medium">{job.company}</span>
+                      </div>
+                    </div>
+                    <JobDropdown />
+                  </CardHeader>
+                  <CardContent className="space-y-3">
+                    <div className="flex items-center gap-2 text-sm text-gray-600">
+                      <MapPin className="h-4 w-4" />
+                      <span>{job.location}</span>
+                    </div>
+                    <div className="flex items-center gap-2 text-sm font-semibold text-[#FF5331]">
+                      <DollarSign className="h-4 w-4 text-green-600" />
+                      <span className="text-green-600">{job.pay}</span>
+                    </div>
+                    <div className="flex items-center gap-2 text-xs text-gray-500">
+                      <Clock className="h-3 w-3" />
+                      <span>Posted via {job.origin}</span>
+                    </div>
+                    <div className="mt-3 flex flex-wrap gap-2">
+                      {job.tags.map((tag) => (
```

Comment:
We should try to keep JavaScript logic (like `map`) outside of the main JSX return when at all possible. This helps keep the layout of the page or component easier to understand at a glance. Usually when I want to render a list of items I move the `map` to it's own function and return it and use the function in the JSX.